src/tests/suite/daos_rebuild_simple.c

```diff
@@ -638,6 +638,49 @@ rebuild_objects(void **state)
 	reintegrate_single_pool_target(arg, ranks_to_kill[0], tgt);
 }
 
+static void
+rebuild_sx_object(void **state)
+{
+	test_arg_t	*arg = *state;
+ 	daos_obj_id_t	oid;
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs